main.py

```diff
@@ -364,6 +378,18 @@ def update_game(self):
             self.bullet_last_held_long = False
             self.continous_bullets_message = False
 
+        if pyxel.btnr(pyxel.KEY_Z):
+            if self.used_special_move: return
+            self.used_special_move = True
+            # some cool animations
+            for enem in self.enemies:
+                if enem.is_alive and enem.show:

```

Comment:
It seems to me that `is_alive` was actually `alive`?

```suggestion
                if enem.alive and enem.show:
```